docs/BuildFromSource.md

```diff
@@ -255,3 +265,13 @@ These are available in the [Visual Studio Preview](https://www.visualstudio.com/
 ## Resx files
 
 If you need to make changes to a .resx file, run `dotnet msbuild /t:Resx <path to csproj>`. This will update the generated C#.
+
+## Building installers
+
+For building installers run
+
+```ps1
+build.cmd -BuildInstallers
```

Comment:
Add what you have above here?

Reply:
Or, change the above to just reference this section. This section would contain all the details.

Reply:
Yeah. Sorry, miss that. Will do

Reply:
build.cmd can be replaced with build.ps1 when using powershell and it seems to work (except building installers fail now since after this was merged locally for some reason with zip stuff).